nomad/client_csi_endpoint_test.go

```diff
@@ -141,11 +141,11 @@ func TestClientCSI_NodeForControllerPlugin(t *testing.T) {
 	node3 := mock.Node()
 	node3.ID = uuid.Generate()
 
-	err := state.UpsertNode(1002, node1)
+	err := state.UpsertNode(structs.MsgTypeTestSetup, node1, 1002)
```

Comment:
Noticed that UpsertNode has a different parameter ordering than UpsertJob/UpsertAllocs.  Would be nice to have them consistent.

Reply:
ah thanks, fixed